src/main/java/org/apache/commons/compress/parallel/FileBasedScatterGatherBackingStore.java

```diff
@@ -23,21 +23,26 @@
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.nio.file.Files;
+import java.nio.file.Path;
 
 /**
- * ScatterGatherBackingStore that is backed by a file.
+ * ScatterGatherBackingStore that is backed by a file or path.
  *
- * @since 1.10
+ * @since 1.22
  */
 public class FileBasedScatterGatherBackingStore implements ScatterGatherBackingStore {
-    private final File target;
+    private final Path target;
     private final OutputStream os;
     private boolean closed;
 
     public FileBasedScatterGatherBackingStore(final File target) throws FileNotFoundException {
+        this(target.toPath());
+    }
+
+    public FileBasedScatterGatherBackingStore(final Path target) throws FileNotFoundException {
```

Comment:
This new eleemnt needs a Javadoc comment and since tag.

Reply:
I changed in place the javadoc at class level, do you want me to put it at the constructor level?
```java
/**
 * ScatterGatherBackingStore that is backed by a file or path.
 *
 * @since 1.22
 */
```

Reply:
Reverted change to class level javadoc, added a new one at constructor level with since tag.